waku/waku_archive/driver.nim

```diff
@@ -45,6 +45,15 @@ method getMessages*(driver: ArchiveDriver,
 method getMessagesCount*(driver: ArchiveDriver):
                          Future[ArchiveDriverResult[int64]] {.base, async.} = discard
 
+method getPagesCount*(driver: ArchiveDriver):
+                         Future[ArchiveDriverResult[int64]] {.base, async.} = discard
+
+method getPagesSize*(driver: ArchiveDriver):
+                         Future[ArchiveDriverResult[int64]] {.base, async.} = discard
+
+method performsVacuum*(driver: ArchiveDriver):
```

Comment:
```suggestion
method performVacuum*(driver: ArchiveDriver):
```

Nitpick. :) (function names should be verbs in imperative form, i.e. without the `s`)

Reply:
good catch!